clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -181,56 +156,71 @@ public HostLevelIngestionStats(
       Map<String, StoreIngestionTask> ingestionTaskMap,
       Time time) {
     super(metricsRepository, storeName);
-    this.time = time;
```

Comment:
What does this time object do here? I check the master branch code, we passed in the time, set it as local varibable, and in below we don't use it for any purpose.